scripts/.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto eol=lf
+*.bat text=auto eol=crlf
```

Comment:
I think we recently added `.ps1` for quick start. I wonder if we should migrate all `.bat`s to `.ps1` maybe? Abstractly, I think it's better to use one scripting langauge per system, and `.ps1` feels like what _should_ be used on windows. Maybe there are _specific_  reasons to stick with `.bat` though!

Reply:
+1 for moving to `ps1`. I believe PowerShell has been supported by default on Windows since 2012!

Reply:
Good call also re: one scripting language per system.